apps/docs/content/guides/functions/import-maps.mdx

```diff
@@ -130,6 +130,31 @@ import process from 'node:process'
 
 Learn more about npm specifiers and Node built-in APIs in [Deno's documentation](https://docs.deno.com/runtime/manual/node/npm_specifiers).
 
+### Importing from Private Registries
+
+<Admonition type="info">
+
+This feature requires Supabase CLI version 1.207.9 or higher.
+
+</Admonition>
+
+Create a `.npmrc` file within `superbase/functions`. This will allow you to import the private packages into multiple functions. Alternatively, you can place the `.npmrc` file directly inside `supabase/functions/function-name` directory.)
```

Comment:
```suggestion
Create a `.npmrc` file within `supabase/functions`. This will allow you to import the private packages into multiple functions. Alternatively, you can place the `.npmrc` file directly inside `supabase/functions/function-name` directory.)
```